daemon/algod/api/server/v2/account.go

```diff
@@ -0,0 +1,345 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package v2
+
+import (
+	"errors"
+
+	"github.com/algorand/go-algorand/crypto"
+	"github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated"
+	"github.com/algorand/go-algorand/data/basics"
+)
+
+// AccountDataToAccount converts basics.AccountData to v2.generated.Account
+func AccountDataToAccount(
+	address string, record *basics.AccountData, assetsCreators map[basics.AssetIndex]string,
+	lastRound basics.Round, amountWithoutPendingRewards basics.MicroAlgos,
+) (generated.Account, error) {
+
+	assets := make([]generated.AssetHolding, 0)
```

Comment:
nit: 0 is unnecessary, alternatively you could use `len(record.Assets)`

Reply:
done